apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -1006,111 +1004,24 @@ describe('progressReduxTest', () => {
       ]
     });
 
-    it('returns a single category if all lessons have the same category', () => {
+    it('returns lesson group', () => {
       const state = {
-        stages: [
-          fakeLesson('Content', null, 'lesson1', 1),
-          fakeLesson('Content', null, 'lesson2', 2),
-          fakeLesson('Content', null, 'lesson3', 3)
-        ],
-        levelProgress: {},
-        focusAreaStageIds: []
-      };
-
-      const groups = groupedLessons(state);
-      assert.equal(groups.length, 1);
-      assert.equal(groups[0].group, 'Content');
-    });
-
-    it('groups non-adjacent lessons by category', () => {
-      const state = {
-        stages: [
-          fakeLesson('cat1', null, 'lesson1', 1),
-          fakeLesson('cat2', null, 'lesson2', 2),
-          fakeLesson('cat1', null, 'lesson3', 3)
-        ],
-        levelProgress: {},
-        focusAreaStageIds: []
-      };
-
-      const groups = groupedLessons(state);
-      assert.equal(groups.length, 2);
-      assert.equal(groups[0].group, 'cat1');
-      assert.equal(groups[1].group, 'cat2');
-      assert.equal(groups[0].levels.length, 2);
-      assert.equal(groups[1].levels.length, 1);
-      assert.deepEqual(groups[0].lessons, [
-        {
-          name: 'lesson1',
-          id: 1,
-          isFocusArea: false
-        },
-        {
-          name: 'lesson3',
-          id: 3,
-          isFocusArea: false
-        }
-      ]);
-      assert.deepEqual(groups[1].lessons, [
-        {
-          name: 'lesson2',
-          id: 2,
-          isFocusArea: false
-        }
-      ]);
-    });
-
-    it('includes bonus levels if includeBonusLevels is true', () => {
-      const bonusLevel = {
-        ids: [2106],
-        title: 1,
-        bonus: true
-      };
-      const state = {
-        stages: [
-          {
-            flex_category: 'Content',
-            levels: [bonusLevel],
-            lessons: []
-          }
-        ],
-        levelProgress: {},
-        focusAreaStageIds: []
-      };
-
-      let groups = groupedLessons(state, false);
-      assert.equal(groups.length, 1);
-      assert.equal(groups[0].levels.length, 1);
-      assert.equal(groups[0].levels[0].length, 0);
-
-      groups = groupedLessons(state, true);
-      assert.equal(groups.length, 1);
-      assert.equal(groups[0].levels.length, 1);
-      assert.equal(groups[0].levels[0].length, 1);
-      assert.equal(groups[0].levels[0][0]['bonus'], true);
-    });
-
-    it('returns lesson group if experiment is enabled', () => {
-      sinon.stub(experiments, 'isEnabled').returns(true);
-      const state = {
-        stages: [fakeLesson('Flex Category', 'Lesson Group', 'lesson1', 1)],
```

Comment:
why are these tests getting removed?

Reply:
These are the tests specific to grouping by flex categories and I made corresponding ones for lesson groups in past PRs (which are just below them in this file). These tests will fail with this update if not removed